aloha-core/src/main/scala/com/eharmony/aloha/algebra/NaturalOptionTransformation.scala

```diff
@@ -0,0 +1,10 @@
+package com.eharmony.aloha.algebra
+
+import scala.language.higherKinds
+
+/**
+  * Created by ryan on 1/25/17.
+  */
+trait NaturalOptionTransformation[F[_], G[_]] {
```

Comment:
If I understand this correctly, this is to convert one higher kinded type to an option of another higher kinded type.  Why is it needed?